packages/common/src/sandbox/modules.ts

```diff
@@ -251,14 +252,30 @@ export const isMainModule = (
 export const findMainModule = (
   modules: Module[],
   directories: Directory[],
-  entry: string = 'index.js'
+  entry: string = 'index.js',
+  template?: TemplateType
 ) => {
   try {
+    // first attempt: try loading the entry file if it exists
     const module = resolveModule(entry, modules, directories);
-
     return module;
   } catch (e) {
-    return modules[0];
+    try {
+      // second attempt: try loading the first file that exists from
+      // the list of possible defaults in the template defination
+      const templateDefinition = getTemplateDefinition(template);
+
+      const defaultOpenedFiles = templateDefinition.getDefaultOpenedFiles({});
+
+      const defaultOpenModule = defaultOpenedFiles
```

Comment:
Same

Reply:
Can, but wanted to use the variable name as an excuse to label the value

Reply:
I think ESLint will complain too if I'm not mistaken

Reply:
Doesn't look like it 😇

Reply:
Hmm yes I like this clear variable name, good documentation.